docs/rsdl/rapid-pro-rsdl-intro.md

```diff
@@ -15,135 +15,136 @@ to envision, create, and consume a Web API that is compatible with the OData Sta
 
 RAPID APIs are defined by a schema, which can easily be described through RSDL.
 
-### Defining a Type
+### Defining a Structured Type
+
 Let's say that we wanted our API to deal with information about a company. We could describe the properties of a company as follows:
 
-~~~rsdl
+```rsdl
 type Company
```

Comment:
The keyword type is too generic.  It covers primitive, enum, structure, and classes. Instead of 'type', I suggest using 'struct' or 'entitytype' or...

Reply:
Discussed 2020-10-29.

We prefer "type" as simpler and more intuitive, even if not as absolutely correct.